addons/forum-copy-code/addon.json

```diff
@@ -15,14 +15,7 @@
       "runAtComplete": false
     }
   ],
-  "userstyles": [
-    {
-      "url": "userstyle.css",
-      "matches": ["https://scratch.mit.edu/discuss/topic/*"]
-    }
-  ],
```

Comment:
always use userstyles instead of js style injection

Reply:
but if i use a userstyle, it will get dynamic enabled/disabled and you can sometimes see unstyled stuff for a split second on some browsers

Reply:
but you shouldn't use inline styles, thats the point of userstyles in scratch addons lol

Reply:
okay fine.
![ugly](https://user-images.githubusercontent.com/68464103/127770880-2fc1c6e0-60b7-4ad5-866a-71ba1ae91ceb.gif)